examples/pkg/listers/example/v1/testtype.go

```diff
@@ -0,0 +1,172 @@
+//go:build !ignore_autogenerated
+// +build !ignore_autogenerated
+
+/*
+Copyright The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by kcp code-generator. DO NOT EDIT.
+
+package v1
+
+import (
+	apimachinerycache "github.com/kcp-dev/apimachinery/pkg/cache"
+	examplev1 "github.com/kcp-dev/code-generator/examples/pkg/apis/example/v1"
+	"github.com/kcp-dev/logicalcluster"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/tools/cache"
+)
+
+// TestTypeLister helps list testType.
+// All objects returned here must be treated as read-only.
+type TestTypeClusterLister interface {
+	// List lists all testType in the indexer.
+	// Objects returned here must be treated as read-only.
+	List(selector labels.Selector) (ret []*examplev1.TestType, err error)
+
+	// Cluster returns an object that can list and get testType from the given logical cluster.
+	Cluster(cluster logicalcluster.Name) TestTypeLister
+
+	// Note(kcp): Workspace-capable Lister implementation doesn't support support expansions.
+	// TestTypeListerExpansion
+}
+
+// testTypeClusterLister implements the TestTypeClusterLister interface.
+type testTypeClusterLister struct {
+	indexer cache.Indexer
+}
+
+// NewTestTypeClusterLister returns a new TestTypeClusterLister.
+func NewTestTypeClusterLister(indexer cache.Indexer) TestTypeClusterLister {
+	return &testTypeClusterLister{indexer: indexer}
+}
+
+// List lists all testType in the indexer.
+func (s *testTypeClusterLister) List(selector labels.Selector) (ret []*examplev1.TestType, err error) {
+	err = cache.ListAll(s.indexer, selector, func(m interface{}) {
+		ret = append(ret, m.(*examplev1.TestType))
+	})
+	return ret, err
+}
+
+// Cluster returns an object that can list and get testType.
+func (s *testTypeClusterLister) Cluster(cluster logicalcluster.Name) TestTypeLister {
+	return &testTypeLister{indexer: s.indexer, cluster: cluster}
+}
+
+// TestTypeLister helps list testType.
+// All objects returned here must be treated as read-only.
+type TestTypeLister interface {
+	// List lists all testType in the indexer.
+	// Objects returned here must be treated as read-only.
+	List(selector labels.Selector) (ret []*examplev1.TestType, err error)
+	// TestTypes returns an object that can list and get testType.
+	TestTypes(namespace string) TestTypeNamespaceLister
+	// Note(kcp): Workspace-capable Lister implementation doesn't support support expansions.
+	// TestTypeListerExpansion
```

Comment:
Would we be leaving this commented, or just add the comment on the type definition and remove `Expansion` ?